AppCenterAuth/AppCenterAuth/Internals/Ingestion/MSAuthConfigIngestion.m

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#import "MSIdentityConfigIngestion.h"
+#import "MSAuthConfigIngestion.h"
```

Comment:
We probably need to reorder here.

Reply:
I could see couple of files like that. May need to run the formater.

Reply:
This is the right order. The header file for the class should be at the top.

Reply:
Ah the formater does this. Though it's flaky. Anyway resolved.